### DISCUSSION TITLE
A seemingly good product, but the guide is exceptionally poor.

### DISCUSSION THREAD

**Discussion by JoeZing** [hoppscotch/hoppscotch#3962] [General]
Has anyone the same situation ?

**Comment by thetronjohnson** [comment]
@JoeZing Hi are you referring to the documentation?

**Reply by JoeZing** [reply]
Of course,such as I install it use AIO in docker, the Admin dashboard cannot open (404),and I find the same question in issue for a long time still no slove https://github.com/hoppscotch/hoppscotch/issues/3349#issuecomment-1782389992
